pkg/logutil/logging.go

```diff
@@ -186,3 +186,11 @@ func Keys(keys [][]byte) zap.Field {
 func ShortError(err error) zap.Field {
 	return zap.String("error", err.Error())
 }
+
+// Any constructs a field that carries an interface{}.
```

Comment:
```suggestion
// Any constructs a redacted field that carries an interface{}.
```

maybe name it `RedactAny`.